src/cli/lineage.ts

```
@@ -633,6 +633,108 @@
   force?: boolean;
 }
 
+async function buildLineageQuery(options: LineageCleanOptions, logger: Logger): Promise<LineageQuery> {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'buildLineageQuery'. Consider to delete it or rename it.

'buildLineageQuery' is defined here:


(lint/suspicious/noRedeclare)